src/main/java/TicketService.java

```diff
@@ -30,7 +32,6 @@ public static void main(String[] args) {
         BusTicketStorage busTicketStorage = new BusTicketStorage(JsonFilePath);
         System.out.println(busTicketStorage.toString());
 
-
         //Task 8
         String url = "jdbc:postgresql://localhost:5432/my_ticket_service_db";
```

Comment:
Do we actually need this hardcoded values here? We have already all config in hibernate.cfg.xml